facefusion/face_helper.py

```diff
@@ -115,6 +123,28 @@ def normalize_bounding_box(bounding_box : BoundingBox) -> BoundingBox:
 	return numpy.array([ x1, y1, x2, y2 ])
 
 
+def transform_points(points : numpy.ndarray[Any, Any], matrix : Matrix) -> numpy.ndarray[Any, Any]:
+	points = points.reshape(-1, 1, 2)
+	points = cv2.transform(points, matrix) # type:ignore[assignment]
+	points = points.reshape(-1, 2)
+	return points
+
+
+def transform_bounding_box(bounding_box : BoundingBox, matrix : Matrix) -> BoundingBox:
```

Comment:
prepare bounding box for xxx? the method does not say too much what it all does

Reply:
with `matrix` bounding box can be scaled, rotated or translated. thats why name transform is given.